pkg/backup/client.go

```diff
@@ -748,8 +755,48 @@ func SendBackup(
 	return nil
 }
 
+// ChecksumMatches tests whether the "local" checksum matches the checksum from TiKV.
+func (bc *Client) ChecksumMatches(local []Checksum) bool {
+	if len(local) != len(bc.backupMeta.Schemas) {
```

Comment:
Perhaps add a log error here is better

Reply:
Yes, firstly I think this error shall never happens. (Since we just checked `json.Unmarshal` in `FastChecksum`) So I just omitted them and print a log, but in fact, add a error return value will be a better slouation.